test/message/nexttick_throw.js

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable no-undef */
```

Comment:
I agree generally to using `no-undef`, but it is pretty common to use undefined globals in tests to trigger an error. With the rule enabled, one would have to research how to configure the linter when writing a test, which I'd like to avoid.

Using a `global` comment is pretty equivalent to disabling the rule, both require additional ugly comments. I think we're fine with just disabling `no-undef` in the test directory.